src/chat.cpp

```diff
@@ -286,13 +282,15 @@ bool Chat::load()
 			channel.name = channelName;
 
 			if (scriptAttribute) {
-				if (scriptInterface.loadFile("data/chatchannels/scripts/" + std::string(scriptAttribute.as_string())) == 0) {
+				if (scriptInterface.loadFile("data/chatchannels/scripts/" + std::string(scriptAttribute.as_string())) ==
```

Comment:
Should be formatted, or `"data/chatchannels/scripts"` should be a [`filesystem::path`](https://en.cppreference.com/w/cpp/filesystem/path)